src/main/resources/interface-specification.yml

```
@@ -6035,6 +6035,97 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /clients/{clientId}:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    delete:
+      description: Deletes a Client
+      tags:
+        - authorization
```

this would change to
```suggestion
        - clients
```
based on the other comment